src/components/QueryFileUploader/utils/checkFileHaveCsvExtension.js

```diff
@@ -0,0 +1,7 @@
+import { getFileExtension } from '@folio/stripes-data-transfer-components';
+
+export const checkFileHaveCsvExtension = file => {
```

Comment:
This case is covered by the other tests. But please, could you add a separated unit-test for this util function?